consul/datadog_checks/consul/consul.py

```diff
@@ -480,7 +482,7 @@ def _submit_service_status(
         # `consul.catalog.nodes_passing` : # of Nodes with service status `passing` from those registered
         # `consul.catalog.nodes_warning` : # of Nodes with service status `warning` from those registered
         # `consul.catalog.nodes_critical` : # of Nodes with service status `critical` from those registered
-        all_service_tags = self._get_service_tags(service, service_tags)
+        all_service_tags = self._get_service_tags(self.named_service_tags, service, service_tags)
```

Comment:
No need to pass an object attribute to an object method
```suggestion
        all_service_tags = self._get_service_tags(service, service_tags)
```